harvesters/src/main/java/org/fao/geonet/kernel/harvest/BaseAligner.java

```diff
@@ -195,4 +196,14 @@ protected Integer getOwnerId(AbstractParams params) {
         return ownerId;
     }
 
+    /**
+     *  To check if the user/group associated with the harvester
+     *  is the current owner/group owner of the existing record
+        * @param recordToOverwrite
+        * @return true if the owner is the same
+     */
+    protected boolean canOverwriteRecord(Metadata recordToOverwrite) {
```

Comment:
Why not using https://github.com/geonetwork/core-geonetwork/blob/master/core/src/main/java/org/fao/geonet/kernel/AccessManager.java#L240 ?

Privileges can be modified on harvested records, so it makes sense to use it also here no ? See Maria's comment https://github.com/geonetwork/core-geonetwork/issues/2613#issuecomment-491184897

Reply:
@fxprunayre, in this way, we would check the user owning the harvester or the user running the harvester?

Reply:
check should verify against the user owning the record, creator of harvester is not relevant here